clients/src/main/resources/common/message/ProduceResponse.json

```diff
@@ -48,10 +48,15 @@
           "about": "The timestamp returned by broker after appending the messages. If CreateTime is used for the topic, the timestamp will be -1.  If LogAppendTime is used for the topic, the timestamp will be the broker local time when the messages are appended." },
         { "name": "LogStartOffset", "type": "int64", "versions": "5+", "default": "-1", "ignorable": true,
           "about": "The log start offset." },
-        { "name":  "ErrorRecords", "type":  "[]int32", "versions": "8+", "default": "", "ignorable":  true,
-          "about":  "The relative offset of records that cause the batch to be dropped"},
+        { "name": "ErrorRecords", "type": "[]RelativeOffsetAndErrorMessage", "versions":  "8+", "default": "", "ignorable": true,
```

Comment:
This default value doesn't make sense. I think you can leave it empty and the default will be an empty array.